zign/api.py

```diff
@@ -18,7 +19,7 @@
 from .oauth2 import ClientRedirectServer
 
 TOKEN_MINIMUM_VALIDITY_SECONDS = 60*5  # 5 minutes
-
+REDIRECT_SERVER_HOST = "127.0.0.1"
```

Comment:
Changing the `redirect_uri` only works when the IAM/OAuth provider also knows about this changed `redirect_uri` (no way we can roll it out at once, due to the distributed nature of CLI tools).

Reply:
ah I see that makes total sense, remote this form being used for redirection, only server.